packages/schema/parse/src/typeInfo/scalar.ts

```diff
@@ -0,0 +1,35 @@
+interface SupportedScalars {
```

Comment:
Instead of duplicating. We can probably write something like
```
export type SupportedScalars = typeof testInstance;
```

Reply:
Ah very good point